packages/tpetra/core/src/Tpetra_Export_def.hpp

```diff
@@ -243,6 +243,13 @@ namespace Tpetra {
     LO numPermutes = 0;
     LO numExports = 0;
 
+    // FENCE REVIEW - CONFIRMED FAILURE
+    //   Testing: This code is exercised by unit tests.
+    //   GTX960:  Fails with CUDA_LAUNCH_BLOCKING=0 and fence removed. TpetraCore_SubmapExportTests_MPI_2
+    //   White:   Not checked since failure already confirmed on GTX960.
+    //   Plan:    Potentially refactor UVM access below and remove fence.
+    //   Notes:
+
```

Comment:
In setupSamePermuteExport() : It isn't to me clear why this needs a fence.  I need to look at this in more detail.